tools/pkg/platforms/centos7/files/build/script.sh

```diff
@@ -15,8 +14,9 @@ git clone ${MONGOOSEIM_REPO} mongooseim
 cd mongooseim
 git checkout ${COMMIT}
 
-VER=$(cat VERSION)
-
+VER=$(./tools/generate_vsn.sh tag)
+COMMITS_COUNT=$(./tools/generate_vsn.sh commits)
+REV=${2:-$((COMMITS_COUNT+1))}
```

Comment:
I'm not sure if we're specific enough here. If the intention is to simplify packaging side branches, then don't we lose information if the final package version is only `mongooseim-$VER-$REV`? In case of `3.0.0-456` there's probably a slim chance that we'll have two branches 456 commits ahead of 3.0.0, but in case of `3.0.0-3` there's a whole lot of possible ambiguity.

[Fedora Versioning Examples](https://fedoraproject.org/wiki/Package_Versioning_Examples) has examples on how to include a version control tag in a package version. Maybe we should take:

```
$ ./tools/generate_vsn.sh describe
3.0.0-236-g7b9afc984
```

and put the third part (`g7b9afc984`) in the package version? My comment also touches line 40 below, which is not directly affected by this PR.